apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-notice.tsx

```diff
@@ -1,69 +0,0 @@
-import { useDispatch } from '@wordpress/data';
-import { __, sprintf } from '@wordpress/i18n';
-import { FC, useEffect } from 'react';
-import { UPGRADE_NOTICE_ID } from './constants';
-import { useHideTemplatePartHint } from './hooks/use-hide-template-part-hint';
-import { usePreviewingTheme } from './hooks/use-previewing-theme';
-import { useSidebarNotice } from './hooks/use-sidebar-notice';
-
-declare global {
-	interface Window {
-		_currentSiteId: number;
-	}
-}
-
-export const LivePreviewUpgradeNotice: FC< {
-	dashboardLink?: string;
-	previewingTheme: ReturnType< typeof usePreviewingTheme >;
-} > = ( { dashboardLink, previewingTheme } ) => {
-	const { createInfoNotice, removeNotice } = useDispatch( 'core/notices' );
-	useHideTemplatePartHint();
-
-	const noticeText = sprintf(
-		// translators: %s: plan name (e.g. Starter, Explorer).
-		__(
-			'Get access to this theme, and a ton of other features, with a subscription to the %s plan.',
```

Comment:
In case of previewing a paid theme, would it be confusing if this notice is not there? The CTA button just says `Upgrade now` without any explanation on why we should upgrade 🤔 

<img width="382" alt="image" src="https://github.com/user-attachments/assets/58a38a67-cad1-4c22-bd88-cdae1b133a8a" />

cc: @jameskoster 

Maybe we can adjust the copy to something like this

<img width="246" alt="image" src="https://github.com/user-attachments/assets/ca8c4481-e124-4ce2-97c5-15cc56235215" />

Reply:
Good point, I'll proceed with this change if James doesn't have any concerns 🙂

Reply:
I guess the user will have entered this flow with the understanding that they're previewing a premium theme, so with that in mind would something shorter work? "Upgrade and activate"?<br><br>Not a big deal, but the label is a little bit long otherwise.